tests/documentation/DocGrammarBearTest.py

```diff
@@ -0,0 +1,75 @@
+from queue import Queue
+import os.path
+import unittest
+import shutil
+
+from coalib.results.Diff import Diff
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import execute_bear
+
+from bears.documentation.DocGrammarBear import DocGrammarBear
+
+try:
+    import language_check
+    language_check
```

Comment:
>this line needed?

Yes as the same is followed in LanguageToolBear. https://github.com/coala/coala-bears/blob/735dda7583d6221d60b147f17b23685633d8e0f9/tests/natural_language/LanguageToolBearTest.py#L12